Doc Ruby Tests (#145)
ignitionrobotics/ign-math: README.md

    sudo apt-get install libeigen3-dev
    ```

> The optional Ruby tests of Ignition Math requires:

Reviewer: ```suggestion
The optional Ruby tests of Ignition Math require:
```

>  * [Ruby](https://www.ruby-lang.org/). Refer to the [Ruby Documentation](https://www.ruby-lang.org/downloads/) for installation instructions. On Ubuntu Systems `apt-get` can be used to install Ubuntu Package `ruby-dev`:

Reviewer: ```suggestion
 * [Ruby](https://www.ruby-lang.org/). Refer to the [Ruby Documentation](https://www.ruby-lang.org/downloads/) for installation instructions. On Ubuntu systems `apt-get` can be used to install Ubuntu Package `ruby-dev`:
```

    ```
    sudo apt-get install ruby-dev
    ```

>   * [Swig](http://www.swig.org/). Refer to the [Swig Documentation](http://www.swig.org/download.html) for installation instructions. On Ubuntu Systems `apt-get` can be used to install Swig:

Reviewer: ```suggestion
  * [Swig](http://www.swig.org/). Refer to the [Swig Documentation](http://www.swig.org/download.html) for installation instructions. On Ubuntu systems `apt-get` can be used to install Swig:
```

    make codecheck
    ```

## Ruby Tests

### Usage

> The C++ classes are available in Ruby code by interfaces files (.i) used by swig to build a C++ extension module.

Reviewer: ```suggestion
The C++ classes are available in Ruby code by interface files (`.i`) used by swig to build a C++ extension module.
```

> The interfaces and Ruby test codes are in `src` folder. To use a C++ class in Ruby you need:

Reviewer: ```suggestion
The interfaces and Ruby test codes are in the `src` folder. To use a C++ class in Ruby you need to:
```

1. Create an interface file describing the class as in Swig and Ruby reference at [The Ruby-to-C/C++ Mapping](http://www.swig.org/Doc1.3/Ruby.html#Ruby_nn11)

2. Include the interface file in `/src/ing_math.i`

3. Create the Ruby file and import the class as in Swig and Ruby reference at [C++ Classes](http://www.swig.org/Doc1.3/Ruby.html#Ruby_nn18)

### Tests

> `make test` already run all tests, including the ones made in Ruby, but you can run an Ruby test individually using

Reviewer: ```suggestion
`make test` already runs all tests, including the ones made in Ruby, but you can run a Ruby test individually using
```
